src/elements/content-sidebar/SidebarContent.js

```diff
@@ -29,7 +39,7 @@ const SidebarContent = ({ actions, children, className, title, elementId, sideba
             role="tabpanel"
             {...rest}
         >
-            <div className="bcs-content-header">
+            <div className={classNames('bcs-content-header', { 'bcs-activity-thread': hasReplies })}>
```

Comment:
The `SidebarContent` component is a generic wrapper and shouldn't be aware of its specific usages. It currently supports a `className` prop that's applied to the parent. Can we do something like this:

```
// ActivitySidebar.js
<SidebarContent className={classNames('bcs-activity', { 'bcs-activity--full': hasReplies })>

// ActivitySidebar.scss
.bcs-activity--full {
    .bcs-content-header {
        margin: 0;
        padding: 0 25px;
    }
}